app/src/main/java/com/example/grutor/Adapters/SubjectAdapter.java

```diff
@@ -43,41 +48,94 @@ public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+                            holder.rlCardImageContainer.setBackground(resource);
+                        }});
+        } else if (holder.title.getText().toString().equals("English")) {
+            String ENGLISH_CARD_URL = "https://images5.alphacoders.com/394/thumb-1920-394862.jpg";
+            Glide.with(context).load(ENGLISH_CARD_URL)
+                    .into(new SimpleTarget<Drawable>() {
+                        @Override
+                        public void onResourceReady(@NonNull Drawable resource, @Nullable com.bumptech.glide.request.transition.Transition<? super Drawable> transition) {
+                            holder.rlCardImageContainer.setBackground(resource);
+                        }});
+        }
+        else if (holder.title.getText().toString().equals("Science")) {
+            String SCIENCE_CARD_URL = "https://cdn.shopify.com/s/files/1/0600/9130/2138/files/home-2-hero-1_2560x.jpg?v=1634081389";
+            Glide.with(context).load(SCIENCE_CARD_URL)
+                    .into(new SimpleTarget<Drawable>() {
+                        @Override
+                        public void onResourceReady(@NonNull Drawable resource, @Nullable com.bumptech.glide.request.transition.Transition<? super Drawable> transition) {
+                            holder.rlCardImageContainer.setBackground(resource);
+                        }});
+        }
+        else if (holder.title.getText().toString().equals("History")) {
+            String HISTORY_CARD_URL = "https://static.kanopy.com/cdn-cgi/image/fit=cover,height=540,width=960/https://static-assets.kanopy.com/video-images/5ed68f3d-6522-4a43-bcee-e24c4e84a8d6.jpg";
+            Glide.with(context).load(HISTORY_CARD_URL)
+                    .into(new SimpleTarget<Drawable>() {
+                        @Override
+                        public void onResourceReady(@NonNull Drawable resource, @Nullable com.bumptech.glide.request.transition.Transition<? super Drawable> transition) {
+                            holder.rlCardImageContainer.setBackground(resource);
+                        }});
+        }
+        else if (holder.title.getText().toString().equals("Government")) {
+            String GOVERNMENT_CARD_URL = "https://free4kwallpapers.com/uploads/originals/2021/03/10/united-states-capitol-building-wallpaper.jpg";
+            Glide.with(context).load(GOVERNMENT_CARD_URL)
+                    .into(new SimpleTarget<Drawable>() {
+                        @Override
+                        public void onResourceReady(@NonNull Drawable resource, @Nullable com.bumptech.glide.request.transition.Transition<? super Drawable> transition) {
+                            holder.rlCardImageContainer.setBackground(resource);
+                        }});
+        }
+        else if (holder.title.getText().toString().equals("Economics")) {
```

Comment:
too many duplicate code here. actually you can define a array use for loop set all the image.